tasks.py

```diff
@@ -44,6 +44,13 @@ def tests(c):
     c.run(f"{VENV_BIN}/pytest", pty=PTY)
 
 
+@task
+def test_coverage(c):
```

Comment:
Any reason not to name this task `coverage`? (Shorter, doesn't cause contention with `inv tests` completion)

Reply:
Sure, changed to "coverage"